src/minibuf.c

```diff
@@ -1857,20 +1857,10 @@ single string, rather than a cons cell whose car is a string.  */)
   return Qnil;
 }
 
-
-DEFUN ("minibuffer-depth", Fminibuffer_depth, Sminibuffer_depth, 0, 0, 0,
-       doc: /* Return current depth of activations of minibuffer, a nonnegative integer.  */)
-  (void)
-{
-  return make_number (minibuf_level);
-}
-
-DEFUN ("minibuffer-prompt", Fminibuffer_prompt, Sminibuffer_prompt, 0, 0, 0,
-       doc: /* Return the prompt string of the currently-active minibuffer.
-If no minibuffer is active, return nil.  */)
-  (void)
+Lisp_Object
+get_minibuffer_prompt (void)
 {
-  return Fcopy_sequence (minibuf_prompt);
+  return Fcopy_sequence(minibuf_prompt);
```

Comment:
Please honor the C code style. The space between the function and parens should remain.

Reply:
sure, sorry. my C's a little rusty

Reply:
No worries. Besides, the GNU formatting is a little odd until you become accustomed to it.